test/setup-globals.js

```diff
@@ -19,6 +19,7 @@ global.window.tinyMCEPreInit = {
 };
 window.requestAnimationFrame = setTimeout;
 window.cancelAnimationFrame = clearTimeout;
+window.matchMedia = () => ( {} );
```

Comment:
I wonder the extent to which we should try to mimic a proper return value here, i.e. [MediaQueryList](https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList). This implementation _could_ be problematic if a library using `matchMedia` attempted to treat the `matches` property on the return object as a boolean (i.e. `false === window.matchMedia( '...' ).matches`). Conversely, the value wouldn't be accurate, so we can't provide a useful value anyways. From what I can tell, this should be fine enough for how TinyMCE is using it:

https://github.com/tinymce/tinymce/blob/114e2d6878c58d6fe452db39353cc177f842d8bd/src/themes/mobile/src/main/js/touch/view/Orientation.js#L22

A more thorough stub might look like:

https://github.com/WickyNilliams/enquire.js/issues/82#issuecomment-26990494

Reply:
Sure. Should we adjust it here to something like that comment?

Reply:
I think it'd be an improvement, though in whatever solution we use, the result will be inaccurate at best. The implementation in the comment could at least avoid some strange behavior that could occur when trying to use properties of the return value (e.g. `addListener` as a noop at least prevents errors calling undefined as a function)